docs/source/json_parser.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+
+ROOT_DIR = os.getcwd()
+SOURCE_DIR = os.path.join(ROOT_DIR, "docs/source/")
+OUT_FILEPATH = os.path.join(SOURCE_DIR, "parsed_info.json")
+info_json_file_path = os.path.join(SOURCE_DIR, "_tmp/", "info.json")
+
+with open(info_json_file_path) as f:
+    info_json_raw = json.load(f)
+
+def dist_str_to_dict(dist: str) -> dict:
```

Comment:
For this function, you can start with an empty dictionary and assign the variables directly. That saves all these temporary variables:

```
output_dict = {}
output_dict["name"] = pkg[0]
...
```

Reply:
I was told to remove a piece of my code that was similar to this, but will gladly add it back, thanks!